### ISSUE TITLE
Should aria-hidden be effected by aria-owns?

### ISSUE THREAD

**Issue by WilcoFiers** [w3c/aria#1818]
The description of aria-hidden has the following text:

> https://www.w3.org/TR/wai-aria-1.2/#aria-hidden
> An element is considered [hidden](https://www.w3.org/TR/wai-aria-1.2/#dfn-hidden) if it, or any of its ancestors are not rendered or have their aria-hidden attribute value set to true.

This would suggest that if I do something like the following, my list would be empty, because the li is hidden.

```html
  <ul aria-owns="li"></ul>
  <div aria-hidden="true">
    <li id="li">Child node</li>
  </div>
```

This doesn't seem to be the case when testing in Chrome and Firefox. There it seems that `aria-owns` is taking priority over `aria-hidden`. Intuitively, that does make sense to me. So then that makes me wonder whether that phrase in aria-hidden should be changed to maybe something like:

> An element is considered [hidden](https://www.w3.org/TR/wai-aria-1.2/#dfn-hidden) if it, or any of its ancestors are not rendered**, or if any of its [Owner Elements](https://www.w3.org/TR/wai-aria-1.2/#dfn-owning-element)** have their aria-hidden attribute value set to true.

**Comment by spectranaut** [comment]
Discussed last meeting: https://www.w3.org/2022/10/13-aria-minutes#t06

**Comment by spectranaut** [comment]
The decision of the working group is to update the spec according to match the behavior of Chrome and Firefox.
